src/NewChain/CliqueGenesis.ts

```diff
@@ -0,0 +1,853 @@
+        },
+        '00000000000000000000000000000000000000ec': {
+          balance: '0x1'
+        },
+        '00000000000000000000000000000000000000ed': {
+          balance: '0x1'
+        },
+        '00000000000000000000000000000000000000ee': {
+          balance: '0x1'
+        },
+        '00000000000000000000000000000000000000ef': {
+          balance: '0x1'
+        },
+        '00000000000000000000000000000000000000f0': {
+          balance: '0x1'
+        },
+        '00000000000000000000000000000000000000f1': {
+          balance: '0x1'
+        },
+        '00000000000000000000000000000000000000f2': {
+          balance: '0x1'
+        },
+        '00000000000000000000000000000000000000f3': {
+          balance: '0x1'
+        },
+        '00000000000000000000000000000000000000f4': {
+          balance: '0x1'
+        },
+        '00000000000000000000000000000000000000f5': {
+          balance: '0x1'
+        },
+        '00000000000000000000000000000000000000f6': {
+          balance: '0x1'
+        },
+        '00000000000000000000000000000000000000f7': {
+          balance: '0x1'
+        },
+        '00000000000000000000000000000000000000f8': {
+          balance: '0x1'
+        },
+        '00000000000000000000000000000000000000f9': {
+          balance: '0x1'
+        },
+        '00000000000000000000000000000000000000fa': {
+          balance: '0x1'
+        },
+        '00000000000000000000000000000000000000fb': {
+          balance: '0x1'
+        },
+        '00000000000000000000000000000000000000fc': {
+          balance: '0x1'
+        },
+        '00000000000000000000000000000000000000fd': {
+          balance: '0x1'
+        },
+        '00000000000000000000000000000000000000fe': {
+          balance: '0x1'
+        },
+        '00000000000000000000000000000000000000ff': {
+          balance: '0x1'
+        },
+      },
+      number: '0x0',
+      gasUsed: '0x0',
+      parentHash: '0x0000000000000000000000000000000000000000000000000000000000000000',
+    }
+
+    // Remove leading `0x` as it should not be present in the genesis addresses.
+    deployer = CliqueGenesis.removeLeading0x(deployer);
+    sealer = CliqueGenesis.removeLeading0x(sealer);
+
+    initialGenesis.alloc[deployer] = { balance: '0x295be96e640669720000000' };
+    initialGenesis.extraData = this.generateSealerExtraData(sealer);
+
+    return initialGenesis;
+  }
+
+  /**
+   * Converts the sealer to a string that is required in the extra data.
+   */
+  private static generateSealerExtraData(sealer: string): string {
+    return `0x0000000000000000000000000000000000000000000000000000000000000000${sealer}0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000`;
+  }
+
+  /**
+   * Convert a given string to a number. Always radix 10.
+   */
+  private static convertToInt(input: string): number {
+    const output = parseInt(input, 10);
+
+    if (output === NaN) {
```

Comment:
I think this will always return false. 
Use:  `isNaN(output)`

Reply:
Thanks, done